src/devices/bosch.ts

```diff
@@ -13,6 +13,29 @@ const ea = exposes.access;
 // Radiator Thermostat II
 const boschManufacturer = {manufacturerCode: 0x1209};
 
+const sirenVolume = {
+    'Low': 0x01,
+    'Medium': 0x02,
+    'High': 0x03,
+};
+
+const sirenLight = {
+    'Only Light': 0x00,
+    'Only Siren': 0x01,
```

Comment:
Please make these, and all the others (e.g. `sirenVolume`) snake case, e.g. `only_siren`